utils/polus-imagej-util/{{cookiecutter.project_slug}}/src/main.py

```diff
@@ -0,0 +1,222 @@
+'''
+This file is autogenerated from an ImageJ plugin generation pipeline.
+'''
+
+from bfio.bfio import BioReader, BioWriter
+import argparse, logging
+import numpy as np
+from pathlib import Path
+import ij_converter
+import jpype, imagej, scyjava
+import typing, os
+
+# Import environment variables
+POLUS_LOG = getattr(logging,os.environ.get('POLUS_LOG','INFO'))
+POLUS_EXT = os.environ.get('POLUS_EXT','.ome.tif')
```

Comment:
I added this in conform to our other plugins. We need to add in the extension conversion to the code below. See the `python-template` for a reference, and ask me if you don't understand what I mean.